README.md

```diff
@@ -0,0 +1,52 @@
+# Tech Docs Template
+
+The Tech Docs Template is a static site generator to build technical documentation with a GOV.UK style for GaaP (Government as a Platform) products.
```

Comment:
It'll probably also cover things like Frontend and Prototype Kit? So maybe 'to build technical documentation for products that Government can use to build services' or something…